### PR TITLE
Python 3.8 and Native Apple Silicon Support

### LINKED ISSUE #649: Python 3.8 and Apple Silicon Support
**Is your feature request related to a problem? Please describe.**

Currently ark-analysis runs through the Intel -> Arm transition layer. This reduces performance and increases installation complexity.

**Describe the solution you'd like**

Bumping to 3.8 will not only match the python versions for toffy and mibi-bin-tools, but also allow for updated packages to be used, making ark Apple Silicon compatible.

**Describe alternatives you've considered**

Bumping Ark to 3.8 and fixing any issues that rise with API changes, as well as testing for Apple Silicon support on M1.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by srivarra**
- bumped packages, skimage.tifffile -> tifffile
- tiffwriter adjustments
- adjusted connectivity in measure.label

**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #649.

**How did you implement your changes**

Fixed API changes with respect to tifffile, and scikit-image. In addition bumped a few packages up for Apple Silicon compatibility.

Updated rtd requirements as well.

**Remaining issues**

- [x] Adjusting documentation, removing Rosetta installation instructions for miniforge.

### CODE REVIEW

File: ark/phenotyping/cell_cluster_utils_test.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from matplotlib.colors import ListedColormap
 import numpy as np
 import pandas as pd
-from skimage.draw import circle
+from skimage.draw import disk
```
Comment (ngreenwald) [inline ark/phenotyping/cell_cluster_utils_test.py:14]: Is this import used anywhere?

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Awesome work, just one import question.

File: ark/phenotyping/cell_cluster_utils_test.py (hunk 1 above)
Comment (srivarra) [reply ark/phenotyping/cell_cluster_utils_test.py:14]: Nope. Will remove it. Should I go through and remove all the other unused import statements?

Comment (ngreenwald) [reply ark/phenotyping/cell_cluster_utils_test.py:14]: Open an issue, we can do it all at once after the workshop as a cleanup

**Review by ngreenwald — APPROVED** [review]

File: ark/phenotyping/cell_cluster_utils_test.py (hunk 1 above)
Comment (srivarra) [reply ark/phenotyping/cell_cluster_utils_test.py:14]: Sounds good.

**Pull request merged**